### ISSUE TITLE
500 error during first attempt

### ISSUE THREAD

**Issue by maelstromchanneled** [iRayanKhan/homebridge-chamberlain#93]
# Explain the issue occuring:
Just added homebridge-chamberlain to QNAP homebridge
Receiving 500 Error at steady tick

# Expected Result
Receive deviceID information so that I can finish config.json setup

# Info
iOS Version: N/A
Plugin Version: 1.6.0
HomeBridge Version: marcoraddatz/homebridge:latest
Node/Npm Version: unknown

# Log Output:
> Error: invalid response, got HTTP 500: {"code":"500.000","message":"An error occurred while processing the request.","description":"Server Error"}                                            
>     at res.text.then.body (/usr/local/lib/node_modules/homebridge-chamberlain/src/api.js:27:15)                                                                                                                                       
>     at <anonymous>                                                                                                                                                                                                                    
>     at process._tickCallback (internal/process/next_tick.js:188:7)

**Comment by iRayanKhan** [comment]
Hi there,
Sorry to see you are facing issues. 

Are you using the latest version if the plugin, and are your details correct in the config?


Thanks,

Rayan.

**Comment by maelstromchanneled** [comment]
I was having a lot of issues with homebridge setup on QNAP at the time.  I tried several different combinations and also the myq2 plugin.  Eventually I got things working with the myq2 plug-in -- most of my issues were on the QNAP network config side.  So, my problems in general are resolved, but I can't say anymore whether chamberlain plugin would have issues.  Will let you know if I pick back up with that plugin in the future.  Thanks for your help!
